### ISSUE TITLE
feature request: slice_volume

### ISSUE THREAD

**Issue by martinschorb** [mobie/mobie-utils-python#67]
Hi,

as just discussed with @tischi , we would love to show the volume character in Figure 2 by a grid of slices.

@constantinpape would it be possible to have a function `create_slice_grid(source,n,axis=0)` that will:

- generate transforms for `n` evenly spread slices across the volume at `axis` that shifts the volume such that the respective slice is at the origin of that axis.
- each of these transform generates individual `sourcenameaftertransform`s
- show a grid of all of these "slices" (i.e. shifted volumes)

That will be just another amazing MoBIE feature...

**Comment by constantinpape** [comment]
Yes, this should be possible. I am working on text now and will give it a try after that.
